tests/utils/ExampleParentTestCase.cfc

```diff
@@ -0,0 +1,20 @@
+component extends="testbox.system.BaseSpec" {
+
+    /**
+     * @beforeEach
+     */
+    function runThisBefore() {
+        variables.counter++;
+    }
+
+    /**
+     * @afterEach
```

Comment:
I updated the test case to use more than one `beforeEach` annotation to show this.